docs/operations/export_api.md

```diff
@@ -0,0 +1,97 @@
+
+```yaml
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: example-export
+spec:
+  tokenSecretRef: example-token
+  source:
+    apiGroup: ""
+    kind: PersistentVolumeClaim
+    name: vm-disk
+```
+
+In this example the PVC name is vm-disk, but it will be different for your Virtual Machine Disk. Note the PVC doesn't need to contain a Virtual Machine Disk, it can contain any content, but the main use case is exporting Virtual Machine Disks. After you post this yaml to the cluster, and the PVC is not in use by another pod, a new export server is created in the same namespace as the PVC. The VirtualMachineExport CR will contain a status with internal and external links to the export service. The internal links are only valid inside the cluster, and the external links are valid for external access. The `cert` field will contain the CA that signed the certificate of the export server for internal links, or the CA that signed the Route or Ingress.
+
+```yaml
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: example-export
+  namespace: example
+spec:
+  source:
+    apiGroup: ""
+    kind: PersistentVolumeClaim
+    name: example-disk
+  tokenSecretRef: example-token
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2022-06-21T14:10:09Z"
+    reason: podReady
+    status: "True"
+    type: Ready
+  - lastProbeTime: null
+    lastTransitionTime: "2022-06-21T14:09:02Z"
+    reason: pvcBound
+    status: "True"
+    type: PVCReady
+  links:
+    external:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: raw
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example-disk/disk.img
+        - format: gzip
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example-disk/disk.img.gz
+        name: example-disk
+    internal:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: raw
+          url: https://virt-export-example-export.example.svc/volumes/example-disk/disk.img
+        - format: gzip
+          url: https://virt-export-example-export.example.svc/volumes/example-disk/disk.img.gz
+        name: example-disk
+  phase: Ready
+  serviceName: virt-export-example-export
+```
+
+If the source PVC is in use by another pod (such as the virt-launcher pod) then the export will remain pending until the PVC is no longer in use.
+
```

Comment:
An example for how to use the created links is welcomed, maybe aexample to use it as source for new dv? and download to laptop?

Reply:
I added an example of how to copy an image from one cluster to another using CDI.